docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -19,331 +19,362 @@ tags:
+   There may be a delay after deleting the API key.
+1. Once successful, the updated API key status will be reflected in the row.
 
-#### Using tcld
+#### Delete a Service Account API key with `tcld`
 
-Use the `tcld apikey delete` command, to delete an API Key.
-The process for deleting an API Key is the same for a User or a Service Account.
+Use the `tcld apikey delete` command to delete an API key.
+The process for deleting an API key is the same for a user or Service Account.
 
-## API Keys for Namespace Authentication {#namespace-authentication}
+```
+tcld login
+tcld apikey delete --id <api-key-id>
+```
 
-A Namespace can be created with API Key authentication enabled as an alternative to mTLS-based authentication.
-To create a Namespace that supports API Key authentication, specify Allow API Key authentication when creating the Namespace.
+## API keys for Namespace authentication {#namespace-authentication}
 
-When creating a new Namespace, a gRPC endpoint is provided based on the authentication method configured for the Namespace.
+Create a Namespace with API key authentication as an alternative to mTLS-based authentication by selecting "Allow API key authentication" during setup.
+The gRPC endpoint format for the Namespace depends on the authentication method:
 
-- For an mTLS connection, the endpoint is in the format `<namespace>.<account>.tmprl.cloud:7233`.
-- For an API Key connection, the endpoint is in the format `<region>.<cloud_provider>.api.temporal.io:7233`
+- For mTLS connections, use `<namespace>.<account>.tmprl.cloud:7233`.
+- For API key connections, use `<region>.<cloud_provider>.api.temporal.io:7233`.
 
-This gRPC endpoint is used in the connection supplied by the user in the Temporal CLI or SDK when connecting to Temporal Cloud using an API Key.
+Use this gRPC endpoint in the Temporal CLI or SDK to connect to Temporal Cloud with an API key.
 
-## Using API Keys {#using-apikeys}
+## Use API keys to authenticate {#using-apikeys}
 
-API Keys can authenticate into Temporal Cloud using the clients:
+Authenticate with Temporal Cloud using API keys with the following clients:
 
-- [Temporal CLI v 0.12 or higher](/cli)
+- [Temporal CLI v0.12 or higher](/cli)
 - [SDKs (latest recommended)](/develop)
-- [Temporal Cloud CLI tcld](/production-deployment/cloud/tcld/index.mdx)
+- [Temporal Cloud CLI `tcld`](/production-deployment/cloud/tcld/index.mdx)
 - [The Cloud Operations API](/production-deployment/cloud/operation-api.mdx)
-- [Temporalʼs Terraform Provider](/production-deployment/cloud/terraform-provider)
+- [Temporal’s Terraform Provider](/production-deployment/cloud/terraform-provider)
 
 ### Temporal CLI
 
-To use your API Key with the Temporal CLI, you can pass in the API Key to each command using the `--api-key` flag or an environment variable (recommended).
+To use your API key with the Temporal CLI, either pass it with the `--api-key` flag for each command or set it as an environment variable (recommended).
```

Comment:
Examples would be nice here, I think (also because this "either... or" sentence is a bit longer as written and that will break it up).

Reply:
Shortened the sentence, added a pseudo example, and made `TEMPORAL_CLOUD_API_KEY` explicit.